guardian/src/main/java/com/auth0/android/guardian/sdk/GuardianAPIClient.java

```diff
@@ -224,12 +224,12 @@ private String createJWT(@NonNull PrivateKey privateKey,
             String headerAndPayload = base64UrlSafeEncode(gson.toJson(headers).getBytes())
                     + "." + base64UrlSafeEncode(gson.toJson(claims).getBytes());
             final byte[] messageBytes = headerAndPayload.getBytes();
-            final Signature signer = Signature.getInstance("SHA256withRSA", "BC");
+            final Signature signer = Signature.getInstance("SHA256withRSA");
```

Comment:
```
Caused by: java.security.NoSuchAlgorithmException: The BC provider no longer provides an implementation for Signature.SHA256withRSA.  
```

Please see https://android-developers.googleblog.com/2018/03/cryptography-changes-in-android-p.html for more details.